changedetectionio/tests/conftest.py

```diff
@@ -11,6 +11,15 @@
 
 global app
 
+# https://loguru.readthedocs.io/en/latest/resources/migration.html#replacing-caplog-fixture-from-pytest-library
+# Show loguru logs only if CICD pytest fails.
```

Comment:
Verbose logs in CICD are suppressed. Only if the pytest fails, the loguru message be printed.

Reply:
So this still needs to be fixed right?

Reply:
It's fixed with the previous commit in this PR(https://github.com/dgtlmoon/changedetection.io/pull/2036/commits/bcb2e7009588bdc404efd3f9412e6b04bce64601).